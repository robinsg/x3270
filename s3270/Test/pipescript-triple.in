"Set(startTls)Set(insertMode)Set(foo)"
Quit
