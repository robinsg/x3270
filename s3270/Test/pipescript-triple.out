{"result":["true","false","Set(): Unknown toggle name 'foo'"],"result-err":[false,false,true],"success":false,"status":"L U U N N 4 24 80 0 0 0x0 0.000"}
L U U N N 4 24 80 0 0 0x0 0.000
ok
